storage/series/series.go

```diff
@@ -706,3 +729,25 @@ func (s *dbSeries) Reset(
 	s.onRetrieveBlock = onRetrieveBlock
 	s.blockOnEvictedFromWiredList = onEvictedFromWiredList
 }
+
+// FlushOutcome is an enum that provides more context about the outcome
+// of series.Flush() to the caller.
+type FlushOutcome int
+
+const (
+	// FlushErr is just a default value that can be returned when we're also returning an error
+	FlushErr FlushOutcome = iota
```

Comment:
nit: could you prefix these to make it clear they're part of the enum, i.e. 

```
type FlushOutcome int 

const (
   FlushOutcomeErr FlushOutcome = iota // ... 
   FlushOutcomeBlockDoesNotExist
   FlushOutcomeFlushedToDisk
) 
```

Reply:
done